apps/web/components/PageWrapper.tsx

```diff
@@ -27,6 +30,23 @@ const calFont = localFont({
   weight: "600",
 });
 
+const Redirects = () => {
```

Comment:
We should either rename it to convey that it is doing redirects related to platform or we can abstract the platform related logic out of this Component.